src/tools/vs-code.md

```diff
@@ -41,56 +41,65 @@ There are a couple ways to create a new project.
 To create a new Flutter project from the Flutter
 starter app template:
 
- 1. Open the Command Palette
-    (`Ctrl`+`Shift`+`P` (`Cmd`+`Shift`+`P` on macOS)).
- 1. Select the **Flutter: New Project** command and press `Enter`.
- 1. Select **Application** and press `Enter`.
- 1. Select a **Project location**.
- 1. Enter your desired **Project name**.
+1. Open the Command Palette.
+
+   - On Windows or Linux,
+     press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>.
+   - On macOS,
+     press <kbd>Cmd</kbd> + <kbd>Shift</kbd>+<kbd>P<kbd>.
+
+1. Tyoe `flutter`.
+1. Select the **Flutter: New Project**.
+1. Press <kbd>Enter</kbd>.
+1. Select **Application**.
+1. Press <kbd>Enter</kbd>.
+1. Select a **Project location**.
+1. Enter your desired **Project name**.
 
 ### Opening a project from existing source code
 
 To open an existing Flutter project:
 
- 1. Click **File > Open** from the main IDE window.
- 1. Browse to the directory holding your existing
-    Flutter source code files.
- 1. Click **Open**.
+1. From the main IDE window, go to **File** > **Open**.
```

Comment:
I know this is pre-existing, but what does "main IDE window mean"? There is only one window in VS code, so it may have been included accidentally.

Perhaps "In the VS Code menu bar ..."?